packages/terra-button-group/package.json

```diff
@@ -44,5 +44,13 @@
     "wdio-lowlight": "cd ../.. && terra wdio --themes clinical-lowlight-theme",
     "wdio-fusion": "cd ../.. && terra wdio --themes orion-fusion-theme",
     "wdio": "npm run wdio-default && npm run wdio-lowlight && npm run wdio-fusion"
-  }
+  },
+  "files": [
```

Comment:
If translations are added to any of the components in future, should those be added in package.json as well for them to work in terra ui?

Reply:
Yes, if any additional folders get added in the future, they will need to be added to this list.